b2/raw_simulator.py

```diff
@@ -15,15 +15,32 @@
 from six.moves import range
 
 from .exception import (
-    BadJson, BadUploadUrl, ChecksumMismatch, Conflict, DuplicateBucketName, FileNotPresent,
-    InvalidAuthToken, MissingPart, NonExistentBucket
+    BadJson,
+    BadUploadUrl,
+    ChecksumMismatch,
+    Conflict,
+    DuplicateBucketName,
+    FileNotPresent,
+    InvalidAuthToken,
+    MissingPart,
+    NonExistentBucket,
+    Unauthorized,
 )
 from .raw_api import AbstractRawApi, HEX_DIGITS_AT_END
 from .utils import b2_url_encode
 
 ALL_CAPABILITES = [
-    'listKeys', 'writeKeys', 'deleteKeys', 'listBuckets', 'writeBuckets', 'deleteBuckets',
-    'listFiles', 'readFiles', 'shareFiles', 'writeFiles', 'deleteFiles'
+    'listKeys',
```

Comment:
a clone, consider `import .account_info.abstract.ALL_CAPABILITIES` instead of redefining it

Reply:
I debated that, but put off doing it because I wasn't sure that the `raw_api` should know about `account_info`.  

If anything, I think that this list belongs in the `raw_api`, and the account_info should use it from there.

Reply:
It should live in one place. I agree that `raw_api` is a better home for it.